lib/vrl/stdlib/src/log_util.rs

```diff
@@ -73,6 +73,44 @@ lazy_static! {
     "#)
     .expect("failed compiling regex for error log");
 
+    // - Nginx HTTP Server docs: http://nginx.org/en/docs/http/ngx_http_log_module.html
+    pub static ref REGEX_NGINX_COMBINED_LOG: Regex = Regex::new(
+        r#"(?x)                                 # Ignore whitespace and comments in the regex expression.
+        ^\s*                                    # Start with any number of whitespaces.
+        (-|(?P<client>\S+))\s+                  # Match `-` or any non space character
+        (-|(?P<user>\S+))\s+                    # Match `-` or any non space character
+        \-\s+                                   # Always a dash
+        \[(?P<timestamp>.+)\]\s+                # Match date between brackets
+        "(?P<request>
+        (?P<method>\w+)\s+                      # Match at least a word
+        (?P<path>\S+)\s+                        # Match any non space character
+        (?P<protocol>\S+)
+        )"\s+                                   # Match any non space character
+        (?P<status>\d+)\s+                      # Match numbers
+        (?P<size>\d+)\s+                        # Match numbers
+        "(-|(?P<referer>.+))"\s+                # Match `-` or any non space character
+        "(-|(?P<agent>.+))"\s+                  # Match `-` or any non space character
+        "(-|(?P<compression>\S+))"              # Match `-` or any non space character
+        \s*$                                    # Match any number of whitespaces (to be discarded).
+    "#)
+    .expect("failed compiling regex for Nginx combined log");
+
+    pub static ref REGEX_NGINX_ERROR_LOG: Regex = Regex::new(
+        r#"(?x)                                         # Ignore whitespace and comments in the regex expression.
+        ^\s*                                            # Start with any number of whitespaces.
+        (?P<timestamp>\d+/\d+/\d+\s\d+:\d+:\d+)\s+      # Match a date with format YYYY/MM/DD hh/mm/ss
```

Comment:
Is the timestamp always this format? Can we make this work the same as the combined log where the user has the option of specifying the format?

Reply:
I don't really know, there's no clear documentation about this error log format. I started nginx in a docker container and triggered an error, that was the result. Maybe I can go dig into nginx code source.

Reply:
I can't find much online either.

Reply:
The definition of the function is made [here](https://github.com/nginx/nginx/blob/1e92a0a4cef98902aed35d7b402a6a402951aba4/src/core/ngx_log.c#L93)

Reply:
And from [this](https://github.com/nginx/nginx/blob/1e92a0a4cef98902aed35d7b402a6a402951aba4/src/core/ngx_times.c#L68) apparently, it's always the same format.

Reply:
Nice.